lib/config/fileconf.go

```diff
@@ -758,6 +762,26 @@ func (b *BPF) Parse() *bpf.Config {
 	}
 }
 
+// RestrictedSession is a configuration for limiting access to kernel objects
+type RestrictedSession struct {
+	// Enabled enables or disables enforcemant for this node.
+	Enabled string `yaml:"enabled"`
+
+	// EventsBufferSize is the size in bytes of the channel to report events
+	// from the kernel to us.
+	EventsBufferSize *int `yaml:"events_buffer_size,omitempty"`
+}
+
+// Parse will parse the enhanced session recording configuration.
+func (r *RestrictedSession) Parse() (*restricted.Config, error) {
+	enabled, _ := apiutils.ParseBool(r.Enabled)
```

Comment:
Why is the error being dropped?

Reply:
Great question, don't remember. I'll fix.